assets/js/boldgrid-inspirations.js

```diff
@@ -1615,19 +1623,28 @@ IMHWPB.InspirationsDesignFirst = function( $, configs ) {
 			self.$themes.empty();
 
 			_.each( self.genericBuilds, function( build ) {
+				build.categoryIsHidden = self.getCategoryHidden( build );
```

Comment:
This variable is only used once. Is it needed?

Maybe change:
`if ( ! build.categoryIsHidden ) {`
to
`if ( ! self.getCategoryHidden( build ) ) {`
?